library/std/src/net/ip.rs

```diff
@@ -485,22 +472,22 @@ impl Ipv4Addr {
     ///
     /// The following return false:
     ///
-    /// - private addresses (see [`is_private()`](#method.is_private))
-    /// - the loopback address (see [`is_loopback()`](#method.is_loopback))
-    /// - the link-local address (see [`is_link_local()`](#method.is_link_local))
-    /// - the broadcast address (see [`is_broadcast()`](#method.is_broadcast))
-    /// - addresses used for documentation (see [`is_documentation()`](#method.is_documentation))
-    /// - the unspecified address (see [`is_unspecified()`](#method.is_unspecified)), and the whole
+    /// - private addresses (see [`Ipv4Addr::is_private`])
+    /// - the loopback address (see [`Ipv4Addr::is_loopback`])
+    /// - the link-local address (see [`Ipv4Addr::is_link_local`])
+    /// - the broadcast address (see [`Ipv4Addr::is_broadcast`])
+    /// - addresses used for documentation (see [`Ipv4Addr::is_documentation`])
+    /// - the unspecified address (see [`Ipv4Addr::is_unspecified`]), and the whole
     ///   0.0.0.0/8 block
     /// - addresses reserved for future protocols (see
-    /// [`is_ietf_protocol_assignment()`](#method.is_ietf_protocol_assignment), except
+    /// [`Ipv4Addr::is_ietf_protocol_assignment`], except
```

Comment:
Ditto with `()` in the other links in this paragraph.
```suggestion
    /// [`Ipv4Addr::is_ietf_protocol_assignment()`], except
```